src/interpreter.rs

```diff
@@ -29,6 +29,16 @@ impl InterpreterConfig {
         self.init_hooks.push(hook);
         self
     }
+    pub fn add_native_module(
```

Comment:
Thank you for a nice advice, really.
I am sorry that I am currently barely maintaining the project and probably not going to finish it in near future.
I am going to merge this PR not because the advice is less useful but just because we eventually need to finish both the code and document. I had to finish at the moment, probably I shared example to the discord chat.